test/setup.js

```diff
@@ -51,5 +51,8 @@ global.document = window.document;
 global.navigator = {
   userAgent: "node.js",
 };
+global.cancelAnimationFrame = (callback = () => {}) => {
+  callback();
+};
```

Comment:
I testene får man `cancelAnimationFrame not defined`, og det føles litt som en liten hack å bare "definere" den som en tom funksjon for at den hvertfall skal finnes.